doc/Type/Metamodel/CoercionHOW.pod6

```diff
@@ -0,0 +1,99 @@
+=begin pod :kind<Type> :subkind<class> :category<metamodel>
+
+=TITLE class Metamodel::CoercionHOW
+
+=SUBTITLE Metaclass for type coercions
+
+I<Warning>: this class is part of the Rakudo implementation, and is not
+a part of the language specification.
+
+In signatures, I<type coercions> may be used to change the types of
+arguments or return values from one type to another:
+
+=begin code
+sub sum-length(Numeric:D(Iterable:D) $a, Numeric:D(Iterable:D) $b --> Numeric:D) {
+    $a + $b
+}
+
+say sum-length [1,2,3], gather { .take for 4..6 }; # OUTPUT: «6␤»
+=end code
+
+These coercions are simply type objects. Since types in Raku are first
+class, type coercions such as this can be used anywhere, just like any
+other type object:
+
+=for code
+say Int:D(Str:D).^name; # OUTPUT: «Int:D(Str:D)␤»
+
+C<Metamodel::CoercionHOW> is the HOW for such coercion type objects.
+Internally, the metamethods this metaclass provides are used to handle
+coercing objects from one type to another.
+
+A type coercion like C<Int:D(Str:D)> may also be written like this:
+
+=for code
+my constant IntStr = Metamodel::CoercionHOW.new_type: Int:D, Str:D;
+say IntStr.^name; # OUTPUT: «Int:D(Str:D)␤»
+
+Coercions L<pretend to be|/type/Metamodel::TypePretense> of types C<Any>
+and C<Mu>, and L<delegate|/type/Metamodel::MethodDelegation> methods to
+C<Any>.
+
+=head1 Methods
+
+=begin comment
+
+This method's documentation would make more sense once archetypes are
+documented.
+
+=head2 method archetypes
+
+    method archetypes()
+
+Returns the archetypes this HOW has. This HOW simply has the I<coercive>
```

Comment:
Um, I'm kind of weary of these calls forward. Wouldn't it make more sense to submit it _once_ those archetypes have been documented?

Reply:
If I did things right, this part is commented out so it can be put in the documentation once those are documented.

Reply:
Yep, but then you have to remember to do that... It's really better if you create an issue.

Reply:
Made one at https://github.com/Raku/doc/issues/3355